cypress/plugins/index.js

```diff
@@ -11,7 +11,28 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
-module.exports = function() {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
-}
+const mongoose = require("mongoose");
+
+// original template code
+// module.exports = function () {
+//   // `on` is used to hook into various events Cypress emits
+//   // `config` is the resolved Cypress config
+// };
+
+module.exports = async (on) => {
+  on("task", {
+    async emptyPosts() {
+      mongoose.connect("mongodb://0.0.0.0/acebook_test");
+      const result = mongoose.connection.collection("posts").deleteMany({});
+      return result;
+    },
+  });
+
+  on("task", {
+    async emptyUsers() {
+      mongoose.connect("mongodb://0.0.0.0/acebook_test");
+      const result = mongoose.connection.collection("users").deleteMany({});
+      return result;
+    },
+  });
+};
```

Comment:
We now also have comments so add:
`module.exports = async (on) => {
  on("task", {
    async emptyPosts() {
      mongoose.connect("mongodb://0.0.0.0/acebook_test");
      const result = mongoose.connection.collection("comments").deleteMany({});
       return result;
    },
  });
};`